addons/l10n_ch/wizard/qr_invoice_wizard.py

```diff
@@ -0,0 +1,88 @@
+     or to see a list of the non-QR/ISR invoices.
+    The non-QR/ISR invoices will have a note logged in their chatter, detailing the reason of the failure.
+    '''
+    _name = 'l10n_ch.qr_invoice.wizard'
+    _description = 'Handles problems occurring while creating multiple QR-invoices at once'
+
+    nb_qr_inv = fields.Integer(default=lambda self: len(self.env.context.get('default_qr_inv_ids')))
+    nb_isr_inv = fields.Integer(default=lambda self: len(self.env.context.get('default_isr_inv_ids')))
+    nb_classic_inv = fields.Integer(default=lambda self: len(self.env.context.get('default_classic_inv_ids')))
+    qr_inv_text = fields.Text(readonly=1)
+    isr_inv_text = fields.Text(readonly=1)
+    classic_inv_text = fields.Text(readonly=1)
+
+    @api.model
+    def default_get(self, fields):
+        res = super().default_get(fields)
+        self.determine_displayed_text(res)
+        return res
+
+    def print_all_invoices(self):
+        '''
+        Triggered by the Print All button
+        '''
+        all_invoices_ids = self.env.context.get('default_all_inv_ids')
+        return self.env.ref('account.account_invoices').report_action(all_invoices_ids)
+
+    def view_faulty_invoices(self):
+        '''
+        Open a list view of all the invoices that could not be printed in the QR nor the ISR format.
+        '''
+        classic_inv_ids = self.env.context.get('default_classic_inv_ids')
+        self.log_error_in_faulty_inv()
+        action = self.env["ir.actions.act_window"]._for_xml_id('account.action_move_out_invoice_type')
+        action['context'] = "{'default_move_type': 'out_invoice', 'create':False}"
+        action['display_name'] = _('Invalid invoices')
+        action.update({
+            'view_mode': 'list,form',
+            'domain': [('id', 'in', classic_inv_ids)],
+        })
+        return action
+
+    def log_error_in_faulty_inv(self):
+        '''
+        Prints the error stopping the invoice from being QR-printed in the invoice's chatter.
+        '''
+        classic_inv_ids = self.env.context.get('default_classic_inv_ids')
+        if classic_inv_ids:
+            for inv_id in classic_inv_ids:
+                inv = self.env["account.move"].browse(inv_id)
+                msg = inv._get_error_to_log()
+                inv.message_post(body=msg, message_type="comment")
+
+    def determine_invoices_text(self, nb_inv, inv_format=None):
+        '''
+        Creates a sentence explaining nb_inv invoices could be printed in the inv_format format.
+        '''
+        if inv_format is None:
+            inv_format = "classic"
+        if nb_inv == 0:
+            return _("No invoice could be printed in the %s format.", inv_format)
+        if nb_inv == 1:
+            return _("One invoice could be printed in the %s format.", inv_format)
+        return str(nb_inv)+_(" invoices could be printed in the %s format.", inv_format)
+
+    def determine_displayed_text(self, res):
```

Comment:
this method is used only once. Move its content into `default_get` directly plz
Also, you should check the requested fields before assigning them